changes/tasks.py

```diff
@@ -524,6 +560,17 @@ def check_momconnect_change_language(self, data_fields, change):
         else:
             return []
 
+    def check_momconnect_change_msisdn(self, data_fields, change):
+        """
+        Ensures that a new msisdn is provided, and that it is a valid msisdn.
+        """
+        if "msisdn" not in data_fields:
```

Comment:
That's a good point. What if we put the ID of the change object on the blob of the address details?